source/common/types/cardano-node.types.js

```diff
@@ -72,7 +72,7 @@ export type CardanoNodeStorageKeys = {
   PREVIOUS_CARDANO_PID: CardanoPidOptions,
 };
 
-export type CardanoNodeProcessNames = 'cardano-node' | 'cardano-node.exe';
+export type CardanoNodeProcessNames = 'cardano-node' | 'cardano-node.exe' | 'jormungandr' | 'jormungandr.exe';
```

Comment:
You can remove `cardano-node`. There will never be a wallet backend called `cardano-node` any more.
It's `cardano-wallet-jormungandr`.